frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -520,6 +522,40 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(firefoxIntegrationAnnouncement);
   }
 
+  const mailingListAnnouncement: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-mailing-list-heading"),
+    snippet: l10n.getString("whatsnew-feature-mailing-list-snippet"),
+    content: (
+      <WhatsNewContent
+        description={l10n.getString(
+          "whatsnew-feature-mailing-list-description"
+        )}
+        heading={l10n.getString("whatsnew-feature-mailing-list-heading")}
+        image={MailingListHero}
+        cta={
+          <a
+            className={styles.cta}
+            href="https://www.mozilla.org/en-US/newsletter/security-and-privacy/"
+            target="_blank"
+          >
+            {l10n.getString("whatsnew-feature-mailing-list-cta")}
+          </a>
+        }
+      />
+    ),
+    icon: MailingListIcon,
+    dismissal: useLocalDismissal(
+      `whatsnew-feature_mailing-list_${props.profile.id}`
+    ),
+    announcementDate: {
+      year: 2023,
+      month: 6,
+      day: 3,
+    },
+  };
+
+  entries.push(mailingListAnnouncement);
```

Comment:
question: Is there any qualifiers for this news item, or should everyone see it?

Reply:
Great question, I was going to bring this up during our morning meeting (standup). I don't have a date for when this should be visible nor, as you put it, qualifiers as for when to show it.

Reply:
This is now behind a flag.